frontend/packages/core-api-client/src/validation/Recipient.ts

```diff
@@ -0,0 +1,30 @@
+import { FieldKind, FormWithRecord, Record } from '../types';
+import { getFieldKind } from '../utils';
+
+export const validateRecipientHierarch = (
```

Comment:
```suggestion
export const validateRecipientHierarchy = (
```
was this a typo?

Reply:
Yeah good catch. I'll do it in a commit rather than the suggestion as I'll need to change it's uses.

Reply:
typo is till there ;)